ENVIRONMENT.rst

```diff
@@ -0,0 +1,26 @@
+Environment Configuration Settings
+==================================
+
+- **ETCD_HOST**: the DNS A record pointing to Etcd hosts.
+- **ETCD_DISCOVERY_DOMAIN**: the DNS SRV record pointing to Etcd hosts.
+- **PGHOME**: filesystem path where to put PostgreSQL home directory (/home/postgres by default)
+- **APIPORT**: TCP port to Patroni API connections (8008 by default)
+- **BACKUP_SCHEDULE**: cron schedule for doing backups via WAL-E (if WAL-E is enabled, '00 01 * * *' by default)
+- **CRONTAB**: anything that you want to run periodically as a cron job (empty by default)
+- **PGROOT**: a directory where we put the pgdata (by default /home/postgres/pgroot). One may adjust it to point to the mount point of the persistent volume, such as EBS.
+- **WALE_TMPDIR**: directory to store WAL-E temporary files. $PGROOT/../tmp by default, make sure it has a few GBs of free space.
```

Comment:
Let's replace $PGROOT with **PGROOT**. The same is with $scope or $SCOPE - let's replace them with **SCOPE**, so that it looks more like a documentation and less like code comments.